app/controllers/verify_controller.rb

```diff
@@ -70,15 +70,6 @@ def confirm_profile_has_been_created
     redirect_to account_url if idv_session.profile.blank?
   end
 
-  def personal_key
```

Comment:
I wouldn't expect this change to be here. Is it from a merge conflict?

Reply:
It was there before. But now not needed.

Reply:
It's called here:

https://github.com/18F/identity-idp/blob/b0c1f934609613e11d8bd78cc2e4b767e24230c7/app/controllers/verify_controller.rb#L45